lib/sycamore/sycamore/tests/unit/connectors/writers/test_common.py

```diff
@@ -0,0 +1,96 @@
+from sycamore.connectors.writers.common import convert_to_str_dict, drop_types, flatten_data
+
+
+def test_flatten_data_happy():
```

Comment:
Would suggest adding / modifying this test to have some strings with a separator in them eg: `"a.."`